aip/general/0135.md

```diff
@@ -132,6 +111,9 @@ rpc DeleteBook(DeleteBookRequest) returns (google.longrunning.Operation) {
 - Both the `response_type` and `metadata_type` fields **must** be specified
   (even if they are `google.protobuf.Empty`).
 
+**Note:** Declarative-friendly resources (AIP-128) **must** use long-running
```

Comment:
Here LRO is a **must** but in the rest of the guidance it seems to be a **should**...intentionally inconsistent?

Reply:
No, that is probably a mistake -- let me check what was intended.

Reply:
Fixed.